suites/apis/rasDRSTest.js

```diff
@@ -46,10 +43,8 @@ BeforeSuite(async ({ I }) => {
 
   // getting the access_token for the test user
   // test user -> cdis.autotest@gmail.com
-  I.cache.ACCESS_TOKEN = await bash.runCommand('gen3 api access-token cdis.autotest@gmail.com');
-  if (process.env.DEBUG === 'true') {
-    console.log(`Access_Token: ${I.cache.ACCESS_TOKEN}`);
-  }
+  I.cache.ACCESS_TOKEN = await bash.runCommand('gen3 api access-token atharvar@uchicago.edu');
```

Comment:
Maybe we could use `ctds.indexing.test@gmail.com` as the user here?

Reply:
we are should use `cdis.autotest@gmail.com` user, as it is our test user. and we would also check what permissions and access this user has in the jenkins in order to execute the test.